sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -844,7 +841,7 @@
           {
             "name": "If-Match",
             "in": "header",
-            "description": "The Etag to be used to determine if the update operation should be carried out",
+            "description": "A string representing a weak ETag for the device twin, as per RFC7232. It determines if the update operation should be carried out.",
```

Comment:
Follow the same format as I specified above.

Reply:
This operation doesn't fit the same criteria as create/update for etags.